src/main/java/com/codeofus/rent_a_park/models/Person.java

```diff
@@ -25,9 +27,11 @@ public class Person {
     String registration;
 
     @OneToMany(mappedBy = "renter", fetch = FetchType.LAZY, cascade = CascadeType.REMOVE)
+    @Transient
```

Comment:
why @Transient and @OneToMany ? One is defining how to save an entity and the other is saying don't save this?